### PR TITLE
Extend the workspace testing fixture.

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR extends the workspace testing fixture with:

- an email within a workspace
- a document within a folder

Jira: https://4teamwork.atlassian.net/browse/NE-124

### CODE REVIEW

File: opengever/testing/fixtures.py [hunk 1]
```diff
@@ -2157,6 +2157,19 @@ def create_workspace(self):
             .with_asset_file('text.txt')
             ))
 
+        create(
```
Comment (njohner) [inline opengever/testing/fixtures.py:2160]: Can you register it, maybe as `workspace_mail`

File: opengever/testing/fixtures.py [hunk 2]
```diff
@@ -2157,6 +2157,19 @@ def create_workspace(self):
...
+            Builder("mail")
+            .with_message(MAIL_DATA)
+            .within(self.workspace)
+            )
+
+        create(
```
Comment (njohner) [inline opengever/testing/fixtures.py:2166]: This one too?

**Review by njohner — CHANGES_REQUESTED** [review]
LGTM, I'd just like to see them registered. I hate objects in the fixture that one cannot access easily. I know it's for the frontend for now, but still.
Also they should appear in the README in the list of fixture objects no?

File: opengever/testing/fixtures.py (hunk 1 above)
Comment (elioschmutz) [reply opengever/testing/fixtures.py]: if you hate it, I'll update it 😄

**Comment by elioschmutz** [comment]
readme updated. It was not required because the new objects have not been registered.

**Review by njohner — APPROVED** [review]
LGTM, thanks!

**Pull request merged**